.github/workflows/coding-standards.yml

```
@@ -27,12 +27,14 @@ jobs:
             - name: Install Composer dependencies
               run: |
                   composer install
-                  composer require rector/rector symplify/easy-coding-standard --dev
+                  mkdir -p tools/php-cs-fixer tools/rector
+                  composer require --working-dir=tools/php-cs-fixer friendsofphp/php-cs-fixer --dev
+                  composer require rector/rector --dev
 
-            - name: CONDING STANDARDS (ECS)
+            - name: CODING STANDARDS (ECS)
```

```suggestion
            - name: PHP-CS-Fixer
```
? (if you have another cs fixer, e.g. for YAML files, you'll probably put it in a dedicated step